beacon-chain/blockchain/metrics_test.go

```diff
@@ -9,16 +9,6 @@ import (
 	"github.com/OffchainLabs/prysm/v6/testing/util"
 )
 
-func TestReportEpochMetrics_BadHeadState(t *testing.T) {
```

Comment:
See https://github.com/OffchainLabs/prysm/pull/15414/files#r2151969948. `SetValidators(nil)` is not an issue because the mvslice can handle it.